src/server/ApiDispatchers.cpp

```diff
@@ -640,6 +644,14 @@
     ex.wAttributes = a->Attributes;
     ex.wPopupAttributes = a->PopupAttributes;
 
+    TraceConsoleAPICallWithOrigin("SetConsoleScreenBufferInfoEx",
+                                  TraceLoggingInt32(a->Size.X, "BufferWidthInChars"),
+                                  TraceLoggingInt32(a->Size.Y, "BufferHeightInChars"),
+                                  TraceLoggingInt32(a->CurrentWindowSize.X, "WindowWidthInChars"),
+                                  TraceLoggingInt32(a->CurrentWindowSize.Y, "WindowHeightInChars"),

```

Comment:
nit 1: These have a different name than the actual fields.

nit 2: Technically these are Int16 right? We can improve the way `COORD`s are formatted in traces if we use `TraceLoggingStruct` here. If we add a new macro like `TraceLoggingConsoleCoord` that abstracts this away, I think it might look pretty neat. Something like
```cpp
#define TraceLoggingConsoleCoord(value, name) \
    TraceLoggingStruct(2, name), \
    TraceLoggingInt32(value.X, "X"), \
    TraceLoggingInt32(value.Y, "Y")
```

Alternatively, we could also use `TraceLoggingInt16FixedArray(&a->CurrentWindowSize, "CurrentWindowSize")`.

nit 3: People depend on our trace logs and we can't change their names? 😟

Reply:
Ugh, these names, types and values were ported directly from the original `tracing.cpp` code. We can definitely change them. I like T.L.Struct.

The only logs people are depending on are `CookedRead` and `ConsoleAttachDetach`. The rest, we can blow up any way we wish.

Reply:
DONE LOOKS GREAT